src/wcs1_0/src/main/java/org/geoserver/wcs/response/Wcs10DescribeCoverageTransformer.java

```diff
@@ -12,6 +12,7 @@
 import java.math.BigDecimal;
```

Comment:
Please review the modification you made in this class. Why you are parsing again the  fixed value string. Here we have the readerDimensionAccessor that should already do what we need. At this point no further actions should be needed since this class is accessing the elevation and time domain values from the reader dimension accessor.